libtfhe/core/arithmetic/random_gen_big.cpp

```diff
@@ -0,0 +1,30 @@
+#include "random_gen.h"
+
+using namespace std;
+
+/**
+ * Instantiate RandomGenTorus class for big torus type
+ */
+template
+class RandomGenTorus<BigTorus>;
+
+void RandomGenTorus<BigTorus>::uniform(
+        TORUS *dst,
+        const ZModuleParams<TORUS> *params) {
+
+    mp_limb_t *const data = dst->data;
+    for (int i = 0; i < params->max_nbLimbs; ++i) {
+        data[i] = RandomGenTorus<Torus64>::uniform(nullptr);
+    }
+}
+
+void RandomGenTorus<BigTorus>::gaussian(
+        TORUS *dst,
+        const TORUS *mean,
+        const double sigma,
+        const ZModuleParams<TORUS> *params) {
+
+    // double precision normal should be fine for big torus
```

Comment:
Add a TODO here + an issue
the double version will provide roughly 53 bits of entropy, that might not be enough in BigTorus apps